Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Pretendo Network/Inkay](https://hosted.weblate.org/projects/pretendonetwork/inkay/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/pretendonetwork/inkay/horizontal-auto.svg)

DaniElectra committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/es/

ashquarky commented:
🔥

DimitriPilot3 committed:
Translated using Weblate (French)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/fr/

DimitriPilot3 committed:
Translated using Weblate (French)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/fr/

Barcia04 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/es/

kaasislekker999 committed:
Translated using Weblate (Dutch)

Currently translated at 92.3% (12 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/nl/

DimitriPilot3 committed:
Translated using Weblate (French)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/fr/

DimitriPilot3 committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/ja/

ashquarky committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/ja/

DimitriPilot3 committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/ja/

DimitriPilot3 committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/ja/

DimitriPilot3 commented:
The CI build failure comes from an incorrect (reversed) ordering of the designator clauses in `fr_FR.lang`, `ja_JP.lang`, and `nl_NL.lang`. `.module_not_found` should appear before `.module_init_not_found`.

For some reason though, `es_ES.lang` has the right ordering. Huh.

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 92.3% (12 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/pt_BR/

EllieNotFound committed:
Translated using Weblate (Italian)

Currently translated at 76.9% (10 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/it/

DoodaPooda committed:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/zh_Hans/

almi05 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/it/

Nervi05 committed:
Translated using Weblate (German)

Currently translated at 92.3% (12 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/de/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/de/

Nervi05 committed:
Translated using Weblate (German)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/de/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/de/

Wohlstand committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/ru/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/es/

kaio486 committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (13 of 13 strings)

Translation: Pretendo Network/Inkay
Translate-URL: https://hosted.weblate.org/projects/pretendonetwork/inkay/pt_BR/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PretendoNetwork/Inkay?pullRequest=54) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PretendoNetwork/Inkay?pullRequest=54) before we can accept your contribution.<br/>**3** out of **11** committers have signed the CLA.<br/><br/>:white_check_mark: DimitriPilot3<br/>:white_check_mark: DaniElectra<br/>:white_check_mark: ashquarky<br/>:x: kaasislekker999<br/>:x: Barcia04<br/>:x: EllieNotFound<br/>:x: Nervi05<br/>:x: almi05<br/>:x: DoodaPooda<br/>:x: kaio486<br/>:x: Wohlstand<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PretendoNetwork/Inkay?pullRequest=54) it.</sub>

DaniElectra committed:
fix(lang): Fix translation entries order
